src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -65,7 +65,7 @@ describe('MdAutocomplete', () => {
 
           return {getContainerElement: () => overlayContainerElement};
         }},
-        {provide: Dir, useFactory: () => ({value: dir})},
+        {provide: Directionality, useFactory: () => ({value: dir})},
```

Comment:
Rather than changing all of these tests to add the `Directionality`, we now have a `MdCommonModule` that all of the components import. You can add `BidiModule` to the imports of the common module.